Convert metric aggs docs runtime fields (#71260)
elastic/elasticsearch: docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc

==== Script

> If you need the cardinality of the combination of two fields you can
> create a <<runtime,runtime field>> combining them and aggregate on that.

Reviewer: ```suggestion
If you need the cardinality of the combination of two fields,
create a <<runtime,runtime field>> combining them and aggregate it.
```
